app/state/reducers/ui/searchReducer.js

```diff
@@ -89,6 +91,40 @@ function searchReducer(state = initialState, action) {
       return state.merge({ unitId: null });
     }
 
+    /*eslint-disable */
+    case 'FILTER_BY_NAME': {
+      let {lang, resources} = action.payload;
+      const sortedResources = _.orderBy( resources, resource=> resource.name[lang] , ['asc'] );
+      return {...state, ['results'] : sortedResources.map( data => data.id ) };
+    }
+
+    case 'FILTER_BY_TYPE': {
+      let {lang, resources} = action.payload;
+      const sortedResources = _.orderBy( resources, resource=> resource.type.name[lang], ['asc'] );
+      /* Lang is static because no data on other language */
+      return {...state, ['results'] : sortedResources.map( data => data.id ) };
```

Comment:
this is the same as:
```suggestion
      return {...state, results : sortedResources.map( data => data.id ) };
```

Reply:
Fixed